crates/core/src/contracts/account.rs

```diff
@@ -54,6 +54,30 @@ pub trait Account<'a, P: Provider + Send + Sync + 'a> {
         // TODO: Remove Manual Decoding
         Ok(vec_felt_to_bytes(bytecode[1..].to_vec()))
     }
+
+    /// Returns the class hash of account implementation of the contract.
+    async fn get_implementation(&self, block_id: &BlockId) -> Result<FieldElement, EthApiError<P::Error>> {
```

Comment:
should we call it `implementation()` to respect our no 'get' policy?

Reply:
Will update 🙂